Lecture3/Task1/product/StockManager.java

```diff
@@ -0,0 +1,15 @@
+package product;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class StockManager {
+
+    public static List<Product> getProductsToStock() {
+        List<Product> products = new ArrayList<>();
+        for (int i = 0; i < Type.values().length; i++) {
```

Comment:
if you want to have all types of guitar in your product list - put  `type`  in `ProductFactory.createProducts` param 
for example 
  ProductFactory.createProducts(list<guiter> (which available on stock),type (type of guitar), amount (how much guitar with this type you want) );

Reply:
done